src/components/form/dt-tags/dt-tags.stories.js

```diff
@@ -105,13 +108,16 @@ function Template(args) {
       options="${JSON.stringify(options)}"
       value="${JSON.stringify(value)}"
       ?disabled=${disabled}
+      ?required=${required}
+      requiredMessage=${requiredMessage}
       icon="${icon}"
       iconAltText="${iconAltText}"
       ?private=${isPrivate}
       privateLabel="${privateLabel}"
       ?allowAdd="${allowAdd}"
       ?loading="${loading}"
       ?saved="${saved}"
+      error=${error}
```

Comment:
Put a ? before this so the attribute doesn't render if there is no value for `error`